test/support/protocol_test_case.rb

```diff
@@ -412,6 +399,9 @@ def attach_to_cdp_server
       sleep 0.001 while @reader_thread.status != 'sleep'
       @reader_thread.run
       INITIALIZE_CDP_MSGS.each{|msg| send(**msg)}
+    end
+
+    def assert_cdp_connected
```

Comment:
I don't think that it's not a good idea to divide this process into a method because it could cause bugs when using frames after calling `attach_to_cdp_server`.

Reply:
Sorry I don't understand this. Can you give me an example scenario?

Reply:
Here is a scenario:
1. call `attach_to_cdp_server`.
2. call something that uses `@crt_frames`
3. `@crt_frames` is not the latest frame because it's always changed every time 'Debugger.paused' is called.

Reply:
I've removed it 👍

Reply:
Thank you.